Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM stakater/base-alpine:3.6
+LABEL maintainer "Hazim <hazim_malik@hotmail.com>"
```

Comment:
@hazim1093 plz change this to Statater Team; and as requested b4 we shouldn't use private email addresses